tests/src/services/tiers.service.test.ts

```diff
@@ -446,4 +581,31 @@ describe('TiersService tests', () => {
       expect(enableVPNTierSpy).not.toHaveBeenCalled();
     });
   });
+
+  describe('Remove VPN access based on user tier', () => {
+    it('When VPN is enabled in the canceled tier user, then a request to remove the VPN access for the user is sent', async () => {
```

Comment:
This is a bit confusing. I assume you wanted to say: 

'When VPN was enabled on the cancelled tier, then the request to disable/remove it is sent'